cfgov/cfgov/urls.py

```diff
@@ -289,6 +300,16 @@
     url(r'^token-provider/', token_provider),
 ]
 
+with flagged_urls('CCDB5_RELEASE') as _url:
+    apiBase = '^data-research/consumer-complaints/api/v1'
+    ccdb5_patterns = [
+        _url(apiBase + '/_suggest', CCDB5.suggest),
+        _url(apiBase + '/(?P<id>[0-9]+)$', CCDB5.document),
+        _url(apiBase, CCDB5.search)
+    ]
+urlpatterns += ccdb5_patterns
```

Comment:
Gotcha; that makes sense. I was just thrown by the separation of the single `flagged_url` and the batch of `flagged_urls`. Is it possible to combine them into `flagged_urls`?